examples/gnrc_networking_subnets/README.md

```diff
@@ -0,0 +1,78 @@
+# Auto-configuration for nested subnets on a (simple) tree topology
+
+This example demonstrates IPv6 subnet auto-configuration for networks on a
+tree topology.
+
+This allows to connect multiple links with individual subnets and route
+between them.
+Each link can have an arbitrary number of hosts, but there can be only
+a single router on each link.
+Routers can have multiple interfaces to connect different downlinks.
+
+![](https://raw.githubusercontent.com/RIOT-OS/RIOT/master/doc/doxygen/src/gnrc_ipv6_auto_subnets_simple.svg)
```

Comment:
IIRC 
```suggestion
![](/RIOT-OS/RIOT/master/doc/doxygen/src/gnrc_ipv6_auto_subnets_simple.svg)
```

should work as well.

Reply:
Not if the file is viewed locally

Reply:
Ah yes. But 
```suggestion
![](../../doc/doxygen/src/gnrc_ipv6_auto_subnets_simple.svg)
```

should also work :-)

Reply:
I thought so too, but at least [grip](https://github.com/joeyespo/grip) does not like it.

Reply:
Then why do we need to honor this?

Reply:
Yea you're right